browser/lib/markdown.js

```diff
@@ -55,13 +55,14 @@ class Markdown {
 
     // Sanitize use rinput before other plugins
     this.md.use(sanitize, {
-      allowedTags: ['iframe', 'input',
+      allowedTags: ['iframe', 'input', 'b',
         'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'h7', 'h8', 'br', 'b', 'i', 'strong', 'em', 'a', 'pre', 'code', 'img', 'tt',
         'div', 'ins', 'del', 'sup', 'sub', 'p', 'ol', 'ul', 'table', 'thead', 'tbody', 'tfoot', 'blockquote',
         'dl', 'dt', 'dd', 'kbd', 'q', 'samp', 'var', 'hr', 'ruby', 'rt', 'rp', 'li', 'tr', 'td', 'th', 's', 'strike', 'summary', 'details'
       ],
       allowedAttributes: {
         '*': [
+          'style',
```

Comment:
After taking some time to reflect, I feel in my heart that this is very tempting but a bad idea. Please see https://github.com/BoostIO/Boostnote/issues/1672#issuecomment-373236851 where I try to explain that it doesn't matter that you cannot do a known xss. style is a known problematic attribute, and a potent target for _new_ exploits.